python/lsst/daf/persistence/butlerLocation.py

```diff
@@ -27,9 +27,54 @@
 
 import lsst.daf.base as dafBase
 
+from collections import namedtuple
 import yaml
 
-from . import iterify
+from . import iterify, doImport
+
+class ButlerComposite(object):
+    componentInfo = namedtuple('componentInfo', 'datasetType subset inputOnly')
+
+    def __init__(self, assembler, disassembler, python, dataId, mapper):
+        """Initializer
```

Comment:
Not required now, but you might think of starting to convert to Numpydoc (https://developer.lsst.io/docs/py_docs.html#documenting-python-apis)